xLights/sequencer/tabSequencer.cpp

```diff
@@ -504,13 +504,21 @@ void xLightsFrame::CheckForValidModels()
 
     logger_base.debug("CheckForValidModels: Remove models that already exist.");
 
-    size_t missingModelCount = 0;
+//    size_t missingModelCount = 0;
+    std::vector<std::string> missingModelCount;
```

Comment:
I'd suggest renaming that variable to maybe missingModels since its not really a count anymore.  Also I'd just delete those 2 lines you commented out.  We can look at history if we need to know what was there.

Reply:
okay, I made those updates.